src/status_im2/contexts/communities/discover/view.cljs

```diff
@@ -34,105 +31,102 @@
                              :tag-label (i18n/label :t/podcasts)
                              :resource  (resources/get-image :podcasts)}]}})
 
-(defn render-other-fn [community-item]
-  (let [item (merge community-item
-                    (get mock-community-item-data :data)
-                    {:featured false})]
-    (if (= @view-type :card-view)
-      [community-card/community-card-view-item item #(rf/dispatch [:navigate-to :community-overview item])]
-      [community-list/communities-list-view-item
-       {:on-press      (fn []
-                         (rf/dispatch [:communities/load-category-states (:id item)])
-                         (rf/dispatch [:dismiss-keyboard])
-                         (rf/dispatch [:navigate-to :community {:community-id (:id item)}]))
-        :on-long-press #(rf/dispatch [:bottom-sheet/show-sheet
-                                      {:content (fn []
+(defn render-fn [featured? width]
+  (fn
+    [community-item]
+    (let [item (merge community-item
+                      (get mock-community-item-data :data)
```

Comment:
yeah I agree, I'm not quite sure how long until we have access to these real data points. I will raise this discussion in the features breakdown today, as maybe it's a good idea to move all of this mock data to a centralised place similar to what you are describing.

Reply:
Thanks @J-Son89. This can save us some time in the future and it makes code review a bit easier (at least for me) because then everyone is on the same page to treat the data flowing as if it was production data.